src/main/java/dk/cngroup/lentils/service/CypherService.java

```diff
@@ -78,4 +81,12 @@ public int getScore(Long cypherId, Long teamId) {
     public Cypher findById(Long cypherId) {
         return cypherRepository.findById(cypherId).get();
     }
+
+    public void deleteById(Long id) {
+        cypherRepository.deleteById(id);
+    }
+    public void deleteAlHintsByCypher(Long id) {
+        Optional<Cypher> cypher = cypherRepository.findById(id);
```

Comment:
Já myslím, že deleteall tady u těchle věcí nebude moc potřeba. No uvidíš jak to bude na webu.
každopádně bych taky kontroloval, jestli Optional obsahuje hodnotu (isPresent) a případě pak volal get().
Nebo na to je myslím, že nějaká atomická lambda metoda. Teď nevím.

Pozn.: Ale jestli ty deleteall talčítka máš už v GUI, klidně to tam nech.

Reply:
Optional řeršeno v nové branch